.vscode/settings.json

```diff
@@ -1,3 +1,13 @@
 {
-  "cSpell.words": ["appspot", "DATOCMS", "firebaseapp", "isvisibility", "Titlebox", "ZRRWYXNV"]
+  "cSpell.words": [
+    "appspot",
```

Comment:
It's fantastic that we are leveraging a plugin to ensure correct spelling 💪

In case of our own code names, we can amend the spelling:
`Titlebox` -> `TitleBox`
`isvisibility` -> `isVisible`
`showModaled` -> `showModal`

And then we will not have to add exceptions here 👌